lib/recase.dart

```diff
@@ -1,36 +1,42 @@
 /// An instance of text to be re-cased.
 class ReCase {
   final RegExp _upperAlphaRegex = new RegExp(r'[A-Z]');
-  final RegExp _symbolRegex = new RegExp(r'[ ./_\-]');
+
+  final symbolMap = {
+    ' ': true,
+    '.': true,
+    '/': true,
+    '_': true,
+    '\\': true,
+    '-': true
+  };
```

Comment:
This doesn't serve any additional value as a map. Can you change it to an array?

Reply:
This map was introduced to make the `recase` faster.
Cause array lookup takes more time then retrieving value from map

Reply:
Are you going to notice the difference, for 6 elements, enough to justify misusing a map?

Reply:
This test states 

```dart
import 'package:recase/recase.dart';

void main() {
  final sw = Stopwatch()..start();
  var result = null;
  const N = 10000;
  for (var i = 0; i < N; i++) {
    result = ReCase('This is-Some_sampleText. YouDig?').titleCase;
  }
  print(
      'done in ${sw.elapsedMilliseconds} (${sw.elapsedMilliseconds / N} ms. per iteration)');
  print('$result');
}
```

Reply:
I think the timing diff is negligible. It's not worth introducing an anti-pattern, in my opinion. So my preference would be an array.

I know the size of the the array is static, but the parsed string could vary in size. The intent of this lib was never to parse documents, even though I think some people will probably try to use it that way. 

The lib's intent to is handle conversion of common naming conventions... the key word being "naming". So it shouldn't be expected to handle massive amounts of text, and to change the casing on such things would be out of scope of the lib, because it's not meant to handle punctuations etc.

So what it boils down to is: I wouldn't expect extreme variations of the processing time, so a diff of 0.0013ms should be fine.

Reply:
Fixed. Thanks for collaboration

Reply:
No problem. I was just thinking last night, maybe a Set would solve both of our concerns. I'd be ok with a Set. I'm not sure how it performs, but I'd be curious if it has a hint of a performance gain, due to all elements being unique.

Reply:
Good point)
I'll test with set

Reply:
Set did the thing!
```
LIST
done in 106 (0.0106 ms. per iteration)

SET
done in 94 (0.0094 ms. per iteration)
```